src/ngx_http_lua_kong_ssl.c

```diff
@@ -175,6 +176,48 @@ ngx_http_lua_kong_get_upstream_ssl_verify(ngx_http_request_t *r,
     return ngx_lua_kong_ssl_get_upstream_ssl_verify(&ctx->ssl_ctx, proxy_ssl_verify);
 }
 
+ngx_flag_t
+ngx_lua_kong_ssl_enable_http2_alpn(ngx_ssl_connection_t *ssl,
```

Comment:
```suggestion
ngx_http_lua_kong_ssl_get_http2_alpn_enabled(ngx_ssl_connection_t *ssl,
```
Propose another name with the necessary "_http_" prefix and replace the verb to show the real meaning